<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMain</class>
 <widget class="QWidget" name="WidgetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSH Conection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Connection Parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Host</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHost">
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>User Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditUserName">
            <property name="text">
             <string>iosef</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPort">
            <property name="text">
             <string>22</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPassword">
            <property name="text">
             <string>123456</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDisconnect">
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
